OptimizelySDK.Net35/OptimizelySDK.Net35.csproj

```diff
@@ -12,8 +12,6 @@
     <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <ReleaseVersion>1.2.1</ReleaseVersion>
-    <SignAssembly>true</SignAssembly>
```

Comment:
are we removing signing?

Reply:
No, we are still signing.  The pack.ps1 does the sn.exe signing.  There seem to be about 4 different ways to do sn.exe signing documented at various MS URL's, but the most reliable approach for us has been sn.exe in pack.ps1 .  We're keeping this.  The other 2 ways we're dropping were redundancies which, unfortunately, do interfere with our E2E testing.